src/charactersheet/viewmodels/common/party_manager/index.html

```diff
@@ -1,19 +1,27 @@
-<div class="collapse add-card" id="party-manager-form">
-  <div class="text-right">
-    <button
-      type="button"
-      class="btn btn-xs btn-link"
-      type="reset"
-      data-toggle="collapse"
-      data-target="#party-manager-form"
-      aria-label="Close"
-    >
-      <span
-        class="fa fa-color fa-times-circle fa-color-hover"
-        aria-hidden="true"
-      ></span>
-    </button>
+<div class="collapse mb-3" id="party-manager-form">
```

Comment:
I think something in this change removed the left and right padding from the form.
![image](https://user-images.githubusercontent.com/7286387/148004275-66232da2-d167-4c69-8e34-23c83bcfdc78.png)

Reply:
I did tweak it a bit and made it look like the others. I added some better padding back in my next commit.